cmd/packer-sdc/internal/fix/cmd.go

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package fix
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/pkg/errors"
+)
+
+const cmdPrefix string = "fix"
+
+// Command is the base entry for the fix sub-command.
+type Command struct {
+	Dir  string
+	Diff bool
+}
+
+// Flags contain the default flags for the fix sub-command.
+func (cmd *Command) Flags() *flag.FlagSet {
+	fs := flag.NewFlagSet(cmdPrefix, flag.ExitOnError)
+	fs.BoolVar(&cmd.Diff, "diff", false, "if set prints the differences a rewrite would introduce.")
+	return fs
+}
+
+// Help displays usage for the command.
+func (cmd *Command) Help() string {
+	var s strings.Builder
+	for _, fix := range availableFixes {
+		s.WriteString(fmt.Sprintf("  %s\t\t%s", fix.name, fix.description))
```

Comment:
I believe you should terminate this helpstring with a `\n` so we can have several not on the same line.

```suggestion
		s.WriteString(fmt.Sprintf("  %s\t\t%s\n", fix.name, fix.description))
```